core/controllers/acl_decorators_test.py

```diff
@@ -3662,15 +3662,13 @@ def setUp(self):
             },
             'change to state 1')
 
-        translation_suggestion = suggestion_services.query_suggestions(
-            [('author_id', self.author_id),
-             ('target_id', self.exploration_id)])[0]
-        question_suggestion = suggestion_services.query_suggestions(
-            [('author_id', self.author_id),
-             ('target_id', 'skill_123')])[0]
-        edit_state_suggestion = suggestion_services.query_suggestions(
-            [('author_id', self.author_id),
-             ('target_id', self.exploration_id)])[1]
+        translation_suggestion = suggestion_services.get_submitted_suggestions(
```

Comment:
Instead, should I fetch all the suggestions by the given author and loop over those suggestions to find suggestions of each type? (Existing one seems cleaner to me and it's working fine)

Reply:
Definitely prefer any approach that doesn't involve looking at the element with index 0 (unless there is guaranteed to be exactly one element in the list). In the latter case, maybe assert that.